azure_functions_worker/mmap_handler/file_accessor_linux.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import mmap
+import urllib.parse
+from typing import Optional
+from .memorymappedfile_constants import MemoryMappedFileConstants as consts
+from .file_accessor import FileAccessor
+
+
+class FileAccessorLinux(FileAccessor):
+    """
+    For accessing memory maps.
+    This implements the FileAccessor interface for Linux.
+    """
+    def open_mem_map(self, map_name: str, map_size: int , access: int) -> Optional[mmap.mmap]:
+        try:
+            fd = self._open_mem_map_file(map_name)
+            mem_map = mmap.mmap(fd.fileno(), map_size, access=access)
+            mem_map.seek(0)
+            return mem_map
+        except Exception as e:
+            # TODO Log Error
+            print(e)
```

Comment:
As we discussed, we can possibly use `logger.error(e, exc_info=True))`, since this message is critical.